cloud/ec2.go

```diff
@@ -949,12 +956,20 @@ func (m *ec2Manager) TerminateInstance(ctx context.Context, h *host.Host, user,
 			"message":       "terminated instance",
 			"user":          user,
 			"host_provider": h.Distro.Provider,
-			"instance_id":   *stateChange.InstanceId,
+			"instance_id":   aws.ToString(stateChange.InstanceId),
 			"host_id":       h.Id,
 			"distro":        h.Distro.Id,
 		})
 	}
 
+	grip.Error(message.WrapError(releaseIPAddressForHost(ctx, m.client, h), message.Fields{
+		"message":        "could not release elastic IP address from host",
+		"provider":       h.Distro.Provider,
+		"host_id":        h.Id,
+		"association_id": h.IPAssociationID,
+		"allocation_id":  h.IPAllocationID,
```

Comment:
Are either of these ID's pointers that need to be `aws.ToString`ed?

Reply:
They're just strings, so it should be fine like this.